src/locales/zh-HK.yaml

```
@@ -77,9 +77,10 @@ app:
     msg:
       confirm: 這將刪除所有文件和文件夾！是否確認？
       not_found: 文件未找到
-      processing: 處理中...
+      processing: 處理中... Transtate by oldcookie
```

I assume this was just a copy&paste issue?

```suggestion
      processing: 處理中...
```